src/components/lowering_replay.js

```diff
@@ -121,7 +122,17 @@ class LoweringReplay extends Component {
       let loweringStartTime = moment(this.state.lowering.start_ts);
       let loweringNow = moment(this.props.event.events[v].ts);
       let loweringElapse = loweringNow.diff(loweringStartTime);
-      return moment.duration(loweringElapse).format("d [days] hh:mm:ss");
+
+      if (this.state.utcScrubber){
+        return moment(loweringNow).toISOString(); // formats scrubber tooltip in UTC ISO8601
```

Comment:
`loweringNow` is already constructed with `moment(...)`, so that seems extraneous here?